### PR TITLE
[NEXUS-8115] Puts mark log permission check at a higher level

### PR DESCRIPTION
**Pull request by djsauble**
Issue: https://issues.sonatype.org/browse/NEXUS-8647

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Log.js [hunk 1]
```diff
@@ -260,7 +260,7 @@ Ext.define('NX.coreui.controller.Log', {
    */
   bindMarkButton: function (button) {
     button.mon(
-        NX.Conditions.isPermitted('nexus:logging:update'),
+        NX.Conditions.isPermitted('nexus:logging:mark', 'nexus:logging:update'),
```
Comment (jdillon) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Log.js]: this probably should just be 'nexus:logging:mark'?

Comment (djsauble) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Log.js]: Seems sane. Fixed.

**Comment by jdillon** [comment]
+1

**Comment by mcculls** [comment]
+1

**Pull request closed without merging**